api/data_test.go

```diff
@@ -189,11 +191,19 @@ func TestCreateNewCacheTime(t *testing.T) {
 				So(responseRecorder.Body.Len(), ShouldEqual, 0)
 			})
 		})
+		Convey("When creating a new cache time with no collection id or release time", func() {
+			request := httptest.NewRequest(http.MethodPut, baseURL+testCacheID, bytes.NewBufferString(validBody))
+			responseRecorder := httptest.NewRecorder()
+			dataStoreAPI.Router.ServeHTTP(responseRecorder, request)
+
+			Convey("Then a new cache time should be created with status code 204 with an empty response body", func() {
+				So(responseRecorder.Code, ShouldEqual, http.StatusNoContent)
+				So(responseRecorder.Body.Len(), ShouldEqual, 0)
```

Comment:
This is certainly giving a 204 response but I can't see that it's checking a new cache time is created.

Reply:
Good spot, have updated the test.